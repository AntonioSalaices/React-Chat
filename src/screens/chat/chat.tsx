interface ChatProps {}

export const Chat: React.FC<ChatProps> = () => {
  return <span>In progress chat</span>;
};

Chat.displayName = 'ChatPage';
