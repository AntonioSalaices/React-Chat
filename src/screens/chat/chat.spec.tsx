import { RenderResult, render } from '@testing-library/react';
import { Chat } from './chat';

describe('<Chat> :: Spec', () => {
  let component: RenderResult;
  beforeEach(() => {
    component = render(<Chat />);
  });
  test('should render correctly', () => {
    const { container } = component;

    expect(container).toMatchSnapshot();
  });
});
