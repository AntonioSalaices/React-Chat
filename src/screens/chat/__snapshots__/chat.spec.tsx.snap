// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chat> :: Spec should render correctly 1`] = `
<div>
  <span>
    In progress chat
  </span>
</div>
`;
